src/core/grabber/include/manipulator/modifier_flag_manager.hpp

```diff
@@ -21,6 +21,7 @@ class modifier_flag_manager final {
     states_[static_cast<size_t>(krbn::modifier_flag::right_option)] = std::make_unique<state>("option", "⌥");
     states_[static_cast<size_t>(krbn::modifier_flag::right_command)] = std::make_unique<state>("command", "⌘");
     states_[static_cast<size_t>(krbn::modifier_flag::fn)] = std::make_unique<state>("fn", "fn");
+    states_[static_cast<size_t>(krbn::modifier_flag::hyper)] = std::make_unique<state>("hyper", "hyper");
```

Comment:
Wouldn't this be better to add a possibility to map a single key to an array of modifiers?